File: src/main/java/aurocosh/divinefavor/common/receipes/serialization/RecipeIngredientInstanceCreator.java
package aurocosh.divinefavor.common.receipes.serialization;

import com.google.gson.InstanceCreator;

import java.lang.reflect.Type;

public class RecipeIngredientInstanceCreator implements InstanceCreator<RecipeIngredient> {
    public RecipeIngredient createInstance(Type type) {
        return new RecipeIngredient();
    }
}

File: src/main/kotlin/aurocosh/divinefavor/common/receipes/serialization/RecipeIngredientInstanceCreator.kt
package aurocosh.divinefavor.common.receipes.serialization

import com.google.gson.InstanceCreator

import java.lang.reflect.Type

class RecipeIngredientInstanceCreator : InstanceCreator<RecipeIngredient> {
    override fun createInstance(type: Type): RecipeIngredient {
        return RecipeIngredient()
    }
}
